File: TLX-Toki/Pascal/ModusTerbesar.pas
var
  ar: array[1..1001] of longint;
  i, a, b, c, modus : longint;
begin
  for i := 1 to 1001 do ar[i] := 0;
  readln(a);
  for i := 1 to a do
  begin
    readln(b);
    ar[b] := ar[b] + 1;
  end;

  c := ar[1];
  modus := 0;
  for i := 2 to 1001 do
    if (ar[i] >= c) then
    begin
      modus := i;
      c := ar[i];
    end;
    writeln(modus);
end.

File: TLX-Toki/CPP/ModusTerbesar.cpp
#include<stdio.h>
using namespace std;
 
int main()
{
 long ar[100001];
 long i,j,k,a,b,c,modus;
 
 for(i = 1;i <= 100000;i++){
	 ar[i] = 0;
 }
 
 scanf("%ld",&a);
 for(k = 1;k <= a;k++){
     scanf("%ld",&b);
	 ar[b]++;
 }
 
 c = ar[1];
 modus = 0;
 for(j = 2; j <= 100000; j++){
	if (ar[j] >= c) {modus = j; c = ar[j];}
 }
 printf("%ld\n",modus);
}